docs/requirements.txt

```diff
@@ -8,6 +8,5 @@ sphinx>=4.0
 sphinx_rtd_theme>0.5
 sphinxcontrib-bibtex
 sphinx-autodoc-typehints>=1.0
-sphinx-copybutton>=0.3
```

Comment:
why remove this? 
the rest lgtm

Reply:
@edgarriba `make build-docs` still requires this no?